content/physical_properties/electrical_conductivity/index.rst

```diff
@@ -3,81 +3,136 @@
+
+.. math::
+  \mathbf{E} = \rho \mathbf{J}
+
+
+Chargeability and Frequency-Dependence
+--------------------------------------
+
+Under the influence of an electric field, free charges (such as ions) flow through materials along the direction of the applied field. For most practical applications, the flow of electrical charges is in phase with the applied electric field. However when ionic charges reach an impermeable barrier, they begin to accumulate. As a result, certain materials can act as capacitors for ionic charges; a phenomenon known as induced polarization. Below are two examples of induced polarization in rocks. On the left, ionic charges accumulate because the pore path is blocked by metallic particles (electrode polarization). On the right, ionic charges accumulate because the pore throat is insufficiently large (membrane polarization).
+
+.. figure:: ./images/fig_polarization_example.png
    :align: center
-   :name: Chargeflow
+   :figwidth: 60%
+   :name: fig_polarization_example
+
+The degree of charge accumulation (capacitance) under the influence of an external electric field is described as **chargeability**. Chargeability is frequently considered a separate physical property from conductivity, although the two are related. For chargeable materials, the constitutive relationship (Ohm's law) becomes frequency-dependent:
+
+.. math::
+  \mathbf{J}(\omega) = \sigma (\omega) \mathbf{E}(\omega) 
+
+A commonly used model for describing frequency-dependent conductivity is the Cole-Cole model:
+
+.. math::
+  \sigma (\omega) = \sigma_{\infty} \Bigg [ 1 - \frac{\eta}{1 + (1-\eta)(i \omega \tau)^C} \Bigg ]
+
+where :math:`\sigma_{\infty}` is the infinite frequency limit, :math:`0 \leq \eta \leq 1` is the **chargeability**, and :math:`\tau` and :math:`0 \leq C \leq 1` define the rate of charge accumulation. After taking the reciprocal, the electrical resistivity is commonly expressed as:
 
-   Flow of charges under an applied electric field.
+.. math::
+  \rho (\omega) = \rho_0 \Bigg [ 1 - \eta \Bigg ( 1 - \frac{1}{1 + (i\omega \tau)^C} \Bigg ) \Bigg ]
+
+where :math:`\rho_0` defines the zero-frequency, or DC, resistivity. Electrical conductivity and resitivity as a function of frequency are illustrated below.
 
-.. note::
 
-    Chargeability is often listed as an independent physical property
-    (such as density, magnetic susceptibility) and we will continue with that
-    in this EM resource. It is however, just an element that is necessary in
-    order to provide a complete description of electrical conductivity.
+.. figure:: ./images/Cole_Cole_example.png
+   :align: center
+   :figwidth: 80%
+   :name: fig_Cole_Cole_example
 
-Applicability
--------------
+   Cole-Cole conductivity (left) and resistivity (right) for :math:`\sigma_\infty` = 2 S/m, :math:`\rho_0` = 1 :math:`\Omega m`, :math:`\eta` = 0.5, :math:`\tau` = :math:`10^{-4}` s and :math:`C` = 0.5.
 
-.. todo::
+Relevance to Geophysical Applications
```

Comment:
I'm going to go with "importance in geophysics" unless you have a problem with that.